pyproject.toml

```diff
@@ -60,9 +60,11 @@ changelog = "https://github.com/stravalib/stravalib/blob/main/changelog.md"
 [project.optional-dependencies]
 build = ["build"]
 tests = [
+  "nox",
   "pytest",
   "pytest-cov",
-  "responses"
+  "responses",
+  "typing-extensions"
```

Comment:
This one is interesting! We just dropped python 3.9 and we don't need support for experimental typing being prototyped in 3.13 as we don't support it yet. so i'm curious why this was added.

Reply:
It was one of the things that FawltyDeps found as being an undeclared dependency. Try running fawltydeps on the main branch and this pops out.

Reply:
I believe you! i wonder if you should report a bug in the fawltydeps repo to better understand why it would suggest this tool for our build. it doesn't fully make sense to me unless we were trying to enforce backward typing compatibility  (for instance perhaps when we did support 3.9 and we wanted to use Unions `|`).  but also it's only happening / being added when running tests which is also odd to me. i might be missing something too but this seems like a bug. 

but i do agree that it's best to just close this pr. 
many thanks for your time here!